frontend/src/routes/+page.svelte

```diff
@@ -1,30 +1,53 @@
-<section class="py-4 md:py-10 text-lg space-y-10">
+<script lang="ts">
+  import Youtube from "$/lib/components/icons/Youtube.svelte";
+  import config from "$lib/config";
+</script>
+
+<section class="space-y-10 py-4 text-lg md:py-10">
   <div class="space-y-4">
-    <h1 class="text-3xl pb-1 font-bold">What is Cipherly?</h1>
-    <p class="">
-      Cipherly is a simple password based encryption and decryption tool.
-    </p>
-    <p class="">
+    <h1 class="pb-1 text-3xl font-bold">What is Cipherly?</h1>
+    <p>Cipherly is a browser-based encryption tool for sharing secrets.</p>
+    <p>
       It uses the browser's built-in <a
         target="_blank"
         href="https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto"
-        class="underline underline-offset-4">SubtleCrypto</a
-      > library to provide encryption and decryption of text using a password. The
-      password is never sent to the server, and the encryption and decryption is
-      done entirely in the browser.
+        class="text-primary underline underline-offset-4">SubtleCrypto</a
+      > library to encrypt and decrypt the secret without ever sending the secret
+      to the server.
     </p>
-    <p class="">This tool is a work in progress.</p>
   </div>
 
   <div class="space-y-4">
-    <h1 class="text-3xl pb-1 font-bold">How can I learn more?</h1>
-    <p class="">
+    <h1 class="pb-1 text-3xl font-bold">How can I learn more?</h1>
+    <p>
       You can follow along on the <a
-        class="text-sky-500"
+        class="text-primary underline underline-offset-4"
         href="https://www.youtube.com/@FriendlyTL"
       >
         Friendly TL YouTube channel!
       </a>
     </p>
+    <div class="flex flex-wrap">
+      {#each config.site.videos as video}
+        <a
+          class="mb-6 mr-6 w-full sm:w-auto sm:max-w-[16rem]"
+          target="_blank"
+          href={video.href}
+          referrerpolicy="no-referrer"
+        >
+          <div
+            class="border-border overflow-hidden rounded border shadow-sm hover:shadow"
```

Comment:
There appears to be a problem with the tailwindcss prettier plugin in VS Code vs the command line. If I save this file in VS Code, the prettier plugin reorders the classes as:
- `border-border overflow-hidden rounded border shadow-sm hover:shadow`
And if I run the prettier command line tool, it reorders the classes as:
- `overflow-hidden rounded border border-border shadow-sm hover:shadow`

Does this happen for you too?

Reply:
Hmm, yeah it happens to me as well, I cannot seem to know why or have a solution, we can just remove the `prettier-plugin-tailwindcss` if it's annoying.

Reply:
I had introduced the `prettier-plugin-tailwindcss` in a previous PR and noticed it, and decided that until it's stable between the CLI and VS Code plugin, we'll end up with too much noise in our PRs. So let's remove the plugin for now.